masterbase/models.py

```diff
@@ -1,13 +1,23 @@
 """Module of pydantic models."""
 
+from enum import Enum
+
 from pydantic import BaseModel
 
 
+class ReportReason(str, Enum):
+    """Valid reasons for reports."""
+
+    bot = "bot"
+    cheater = "cheater"
+
```

Comment:
I think more conventional to use SCREAMING_SNAKE_CASE for these attributes so `bot` -> `BOT` and `cheater` -> `CHEATER` but the actual enum values can stay as lowercase

Reply:
done